Gemfile

```diff
@@ -14,7 +14,17 @@ lockfile "rails-7.0" do
   gem "railties", "~> 7.0.0"
 end
 
-lockfile do
+lockfile "rails-7.1" do
```

Comment:
for many tools (including Dependabot), it's important that `Gemfile.lock` exists; please leave one of the variations with that name